apps/omg/lib/omg/utxo/position.ex

```diff
@@ -167,12 +169,18 @@ defmodule OMG.Utxo.Position do
   """
   @spec get_data_for_rlp(t()) :: binary()
   def get_data_for_rlp(Utxo.position(blknum, txindex, oindex)) do
-    utxo = %ExPlasma.Utxo{blknum: blknum, txindex: txindex, oindex: oindex}
-    ExPlasma.Utxo.to_rlp(utxo)
+    utxo = ExPlasmaPosition.new(blknum, txindex, oindex)
+    ExPlasmaPosition.to_rlp(utxo)
   end
 
-  defp do_decode(encoded) do
-    with {:ok, utxo} <- ExPlasma.Utxo.new(encoded),
+  defp do_decode(encoded) when is_binary(encoded) do
+    with {:ok, %ExPlasmaOutput{output_id: %{blknum: blknum, txindex: txindex, oindex: oindex}}} <-
+           ExPlasmaOutput.decode_id(encoded),
+         do: {:ok, Utxo.position(blknum, txindex, oindex)}
```

Comment:
```suggestion
    {:ok, %ExPlasmaOutput{output_id: %{blknum: blknum, txindex: txindex, oindex: oindex}}} = ExPlasmaOutput.decode_id(encoded)
    {:ok, Utxo.position(blknum, txindex, oindex)}
```

Reply:
@achiurizo 
At first I committed this and the following suggestion but we are testing for values returned in case of errors. So I had to revert that.

I have a feeling there is some mess in how returned values are handled here. After I'm done with moving childchain and watcher to explasma from hex, I'll have a look on that. Maybe I could improve something, but for now, let's leave it like it is.